docs/tutorials/essentials/part-4-using-data.md

```diff
@@ -18,7 +18,7 @@ import { DetailedExplanation } from '../../components/DetailedExplanation'
 :::info Prerequisites
 
 - Understanding the [Redux data flow and React-Redux APIs from Part 3](./part-3-data-flow.md)
-- Familiarity with [the React Router `<Link>` and `<Route>` components for page routing](https://reactrouter.com/docs/en/v6/getting-started/overview)
+- Familiarity with [the React Router `<Link>` and `<Route>` components for page routing](https://reactrouter.com/en/6.7.0/start/overview)
```

Comment:
```suggestion
- Familiarity with [the React Router `<Link>` and `<Route>` components for page routing](https://reactrouter.com/start/overview)
```

This link will eventually be outdated. This is always going to work and be up-to-date as long as the page exists.